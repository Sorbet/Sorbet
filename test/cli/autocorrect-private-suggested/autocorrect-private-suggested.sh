# verifies that the private method is suggested as a replacement
tmp="$(mktemp -d)"
infile="test/cli/autocorrect-private-suggested/autocorrect-private-suggested.rb"
cp "$infile" "$tmp"

old_pwd="$(pwd)"
cd "$tmp" || exit 1

"$old_pwd/main/sorbet" --silence-dev-message -a autocorrect-private-suggested.rb 2>&1
