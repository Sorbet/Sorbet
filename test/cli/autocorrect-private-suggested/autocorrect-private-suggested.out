autocorrect-private-suggested.rb:7: Method `my_private_metho` does not exist on `A` https://srb.help/7003
     7 |    my_private_metho
            ^^^^^^^^^^^^^^^^
  Did you mean:
    autocorrect-private-suggested.rb:4: `A#my_private_method`
     4 |  private def my_private_method; end
                  ^^^^^^^^^^^^^^^^^^^^^
Errors: 1
